lnwallet/size_test.go

```diff
@@ -68,41 +68,54 @@ func TestTxWeightEstimator(t *testing.T) {
 		numP2WSHOutputs      int
 		numP2SHOutputs       int
 	}{
+		{},
 		{
-			numP2PKHInputs:  1,
-			numP2PKHOutputs: 2,
+			numP2PKHInputs: 1,
 		},
 		{
-			numP2PKHInputs:  1,
```

Comment:
Looks like combinations of inputs/outputs are not tested anymore? I think that is valuable to keep to some degree, as it ensures the weight estimator handles it.

Reply:
readded tests and also expanded the test suite a good bit